python.d/haproxy.chart.py

```diff
@@ -39,6 +44,34 @@
         'options': [None, "Session In Queue", "sessions", 'frontend', 'haproxy_f.qcur', 'line'],
         'lines': [
         ]},
+    'fhrsp_1xx': {
+        'options': [None, "HTTP responses with 1xx code", "responses", 'frontend', 'haproxy_f.hrsp_1xx', 'line'],
```

Comment:
I think all units should be `responses/s` not `responses` since algorithm is incremental (true for all new charts)